terra_oracle_vote.py

```diff
@@ -737,12 +816,25 @@ def metrics_for_result(exchange, result):
         logger.info("Active set: {}".format(active))
 
         fx_err_flag, real_fx = res_fx.result()
+        fx_band_err_flag, fx_band = res_fx_band.result()
+
+        if not fx_err_flag and not fx_band_err_flag:
+            for key in real_fx:
+                real_fx[key] = (real_fx[key] + fx_band[key])/2.0
```

Comment:
Currently, FX api gives users a choice (free, fcs, alphavantage etc..)

Adding the band's value to this will give the user no choice,

I think users should have a choice of fx provider

What do you think?

Reply:
Or, it would be okay to adopt a weighted selection method like Luna Price.